symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/AuthenticatorFactory.java

```diff
@@ -176,18 +177,18 @@ private PrivateKey loadPrivateKeyFromAuthenticationConfig(BdkAuthenticationConfi
         if (isNotEmpty(config.getPrivateKeyContent())) {
           privateKey = new String(config.getPrivateKeyContent(), StandardCharsets.UTF_8);
         } else {
-          String privateKeyPath = config.getPrivateKeyPath();
+          privateKeyPath = config.getPrivateKeyPath();
           log.debug("Loading RSA privateKey from path : {}", privateKeyPath);
           privateKey = loadPrivateKey(privateKeyPath);
         }
       }
       return JwtHelper.parseRsaPrivateKey(privateKey);
     } catch (GeneralSecurityException e) {
-      final String message = "Unable to parse RSA Private Key";
+      final String message = "Unable to parse RSA Private Key. Check if the format is correct.";
       log.error(message, e);
       throw new AuthInitializationException(message, e);
     } catch (IOException e) {
-      final String message = "Unable to read or find RSA Private Key from path " + config.getPrivateKeyPath();
+      final String message = "Unable to read or find RSA Private Key from path " + privateKeyPath;
       log.error(message, e);
```

Comment:
Yep I would probably remove that log here, otherwise we get 2 stacktraces upon failure

Reply:
done